cocos/core/data/object.ts

```diff
@@ -193,8 +193,8 @@ class CCObject implements EditorExtendableObject {
     /**
      * @internal
      */
-    public _objFlags: number;
-    protected _name: string;
+    public _objFlags: number = 0;
```

Comment:
Why not use declare?

Reply:
Only use `declare` when properties are initialized in `constructor`. Otherwise, it doesn't need to use `declare` keyword.

Reply:
They are initialized in constuctor.